### PR TITLE
Added new parametrized CustomChromium

### PR DESCRIPTION
**Pull request by Mac-Ste**
Added new parametrized CustomChromium allowing to use specified URIs for browser and driver

### CODE REVIEW

File: src/main/kotlin/com/atlassian/performance/tools/infrastructure/api/browser/chromium/CustomChromium.kt [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import com.atlassian.performance.tools.infrastructure.ChromiumInstaller
+import com.atlassian.performance.tools.infrastructure.api.browser.Browser
+import com.atlassian.performance.tools.infrastructure.ParallelExecutor
+import com.atlassian.performance.tools.ssh.api.SshConnection
+import java.net.URI
+
+class CustomChromium(
+    private val chromiumUri: URI,
+    private val chromedriverUri: URI
+) : Browser {
```
Comment (wyrzyk) [inline src/main/kotlin/com/atlassian/performance/tools/infrastructure/api/browser/chromium/CustomChromium.kt]: @Mac-Ste Unfortunately it's not documented, but we require browser implementation to provide a no-args constructor.

See how it's initialized https://github.com/atlassian/virtual-users/blob/master/src/main/kotlin/com/atlassian/performance/tools/virtualusers/LoadTest.kt#L48

Comment (dagguh) [reply src/main/kotlin/com/atlassian/performance/tools/infrastructure/api/browser/chromium/CustomChromium.kt]: @wyrzyk you're confusing `com.atlassian.performance.tools.infrastructure.api.browser.Browser` with the `com.atlassian.performance.tools.virtualusers.api.browsers.Browser`.

Comment (wyrzyk) [reply src/main/kotlin/com/atlassian/performance/tools/infrastructure/api/browser/chromium/CustomChromium.kt]: You're right!

**Comment by dagguh** [comment]
@wyrzyk, when you add a comment as a code owner, you're automatically [reviewing it on behalf of the JPT team](https://github.com/atlassian/infrastructure/pull/58#pullrequestreview-367959675). In general it makes sense, you started reviewing it, it's natural to expect you to follow it further as an owner. However, if you don't plan to push the PR further, then let's [re-add the team as a reviewer](https://github.com/atlassian/infrastructure/pull/60#event-3092611267), so that a maintainer can merge and release it.

**Comment by dagguh** [comment]
@Mac-Ste released in `4.15.0`

**Pull request merged**